cmd/containerboot/main.go

```diff
@@ -142,6 +142,7 @@ func main() {
 		InKubernetes:                          os.Getenv("KUBERNETES_SERVICE_HOST") != "",
 		UserspaceMode:                         defaultBool("TS_USERSPACE", true),
 		StateDir:                              defaultEnv("TS_STATE_DIR", ""),
+		SsmArn:                                defaultEnv("TS_SSM_ARN", ""),
```

Comment:
lets not do this and instead add a `TS_STATE` param which follows the same semantics as the as `--state` field and deprecate `TS_KUBE_SECRET` in favor of that.

Reply:
I initially wondered why that wasn't the way we went forward, definitely agree this is the best approach